### PR TITLE
setup react app

### LINKED ISSUE #7: Setup Create-React-App Project
Create a template codebase using create-react-app. 
- have template files within each new folder created
- install needed dependencies: semantic-ui-react, axios, react-router-dom, Redux, React-Redux, Redux-Thunk
- setup browser router
- setup _redirects file for Netlify
- setup Redux

### PR DESCRIPTION
**Pull request by christopher-alba**
This pull request sets up the react app codebase with a set folder/file structure. All needed dependencies should be installed (check package.json for new dependencies)

- have template files within each new folder created
- install needed dependencies: semantic-ui-react, axios, react-router-dom
- setup browser router
- setup _redirects file for Netlify

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +0,0 @@
-# Frontend
-This repo will be linked to Netlify for ease of deployment and code review
```
Comment (r055a) [inline README.md]: Why did you delete the README.md? I suggest not deleting the README.md to prevent potential git conflict

**Review by hpar461 — DISMISSED** [review]
Upon prior review, a discussion has been made with Chris to use Context API rather than Redux for now. The review has been approved after commits regarding the change has been made.

**Review by R055A — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (christopher-alba) [reply README.md]: oh because we will be replacing it with the new readme.md that's currently in a pull request

Comment (christopher-alba) [reply README.md]: It will prevent a merge conflict later on

Comment (r055a) [reply README.md]: Thank you for the feedback.
If this is merged first, then when you pull the master branch into the new README branch it suspect there will be a conflict. 
If it is merged second, then I suspect it will remove the new README.
It seems unecessary to make changes to the README in this branch when there is already another branch for that.
The README doesn't seem relevant to the React app. The PR description does not include deleting the README

Comment (christopher-alba) [reply README.md]: Ah true that's a good point, I shall revert the readme deletion.

**Review by hpar461 — APPROVED** [review]

**Review by yuchendong13 — APPROVED** [review]
Looks good and works on my machine. Thanks for setting it up.

**Review by R055A — APPROVED** [review]
Looks very good!
